packages/client/src/components/Button/Button.tsx

```diff
@@ -0,0 +1,24 @@
+import { PureComponent }  from 'react';
+import classnames from 'classnames';
+
+import './Button.css';
+
+type ButtonProps = {
+    children: React.ReactNode;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    onClick?: (...args: any) => void;
+    className?: string;
+};
```

Comment:
Лучше ещё добавить `& ButtonHTMLAttributes<HTMLButtonElement>`, чтобы у кнопки были доступны все стандартные атрибуты.

```suggestion
} & ButtonHTMLAttributes<HTMLButtonElement>;
```

А в разметке сделать так: `<button {...props} className={classNames}>`